spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestExpireSnapshotsProcedure.java

```diff
@@ -300,7 +300,7 @@ public void testExpireDeleteFiles() throws Exception {
     Assert.assertEquals("Should have 1 delete file", 1, TestHelpers.deleteFiles(table).size());
     Path deleteManifestPath = new Path(TestHelpers.deleteManifests(table).iterator().next().path());
     DeleteFile deleteFile = TestHelpers.deleteFiles(table).iterator().next();
-    Path deleteFilePath = new Path(String.valueOf(deleteFile.path()));
+    Path deleteFilePath = new Path(String.valueOf(deleteFile.location()));
```

Comment:
Why do we need `String.valueOf` even though `deleteFile.location()` is String?

Reply:
We don't, good catch!